js-webapp/newton.jpg.txt

```diff
@@ -0,0 +1 @@
+https://commons.wikimedia.org/wiki/File:Sir_Isaac_Newton._Wellcome_V0006785EL.jpg
```

Comment:
What is the idea of this file? Will this be alt-text or caption of the image on medium?